Poisson.C

```diff
@@ -375,6 +376,9 @@ PoissonNorm::PoissonNorm (Poisson& p, int integrandType, VecFunc* a) :
 }
 
 
+PoissonNorm::~PoissonNorm() = default;
+
+
```

Comment:
Is there any practical difference between this and an empty destructor? And if yes, why not keep the default declaration in the header file.

Reply:
not as such, but the big difference is that it's the dtor is generated in the source file. in the source file we know the definition of the fields type. hence we avoid pulling in the fields header in the header.